FaunaDB.Client/Types/RefV.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using FaunaDB.Query;
+using Newtonsoft.Json;
+
+namespace FaunaDB.Types
+{
+    /// <summary>
+    /// A FaunaDB ref type.
+    /// <para>
+    /// See <see href="https://fauna.com/documentation/queries#values-special_types">FaunaDB Special Types</see>.
+    /// </para>
+    /// </summary>
+    public class RefV : Value
+    {
+        public string Id { get; }
+        public RefV Class { get; }
+        public RefV Database { get; }
```

Comment:
yes and no, it might be an instance of `RefV` with the same shape of `ClassV`, given `ClassV` is just a sugar, but yes, we can constraint ourselves be a `ClassV` with no harm. I've implemented using `ClassV` and `DatabaseV` in the beginning but for some reason I don't remember now I changed.